crates/ursa-network/Cargo.toml

```diff
@@ -36,6 +36,7 @@ tokio.workspace = true
 tracing.workspace = true
 ursa-metrics = { path = "../ursa-metrics" }
 ursa-store = { path = "../ursa-store" }
+void = "1.0.2"
```

Comment:
is this used anywhere?

Reply:
if it is, should be in the top level cargo

Reply:
oop need to revert, thanks